src/frontend/store/actions/publisherAction.js

```diff
@@ -197,17 +197,30 @@ export const publisherCreationRequest = values => async () => {
   }
 };
 
-export const fetchPublishersRequestsList = ({token}) => async dispatch => {
+export const queryPublishersRequests = ({name, createdBefore, createdAfter, updatedBefore, updatedAfter, backgroundProcessingState, state, createdResource, page, limit, sort, token}) => async dispatch => {
   dispatch(setListLoader());
   try {
     const response = await fetch('/api/requests/publishers/query', {
-      method: 'GET',
+      method: 'POST',
       headers: {
         'Cross-Origin-Opener-Policy': 'same-origin',
         'Cross-Origin-Embedder-Policy': 'require-corp',
         Authorization: `Bearer ${token}`,
         'Content-Type': 'application/json'
-      }
+      },
+      body: JSON.stringify({
+        name: name.length > 0 ? name : undefined,
```

Comment:
Can we remove the "> 0" part?
```suggestion
        name: name.length ? name : undefined,
```

Reply:
0 is falsy so technically yes. However I would in general discourage using falsy interpretations over well-defined boolean tests.